storage/postgres/testdb/testdb.go

```diff
@@ -52,21 +53,44 @@ func PGAvailable() bool {
 	return true
 }
 
+//This just executes a simple query in the configured database.  Only used as a placeholder
+// for testing queries and how go returns results
+func TestSQL(ctx context.Context) string {
+	db, err := sql.Open("postgres", getConnStr(*dbName))
+	if err != nil {
+		fmt.Println("Error: ", err)
+		return "error"
+	}
+	defer db.Close()
+	result, err := db.QueryContext(ctx, "select 1=1")
+	if err != nil {
+		fmt.Println("Error: ", err)
+		return "error"
+	}
+	var resultData bool
+	result.Scan(&resultData)
+	if resultData {
+		fmt.Println("Result: ", resultData)
+	}
+	return "done"
+}
+
 // newEmptyDB creates a new, empty database.
 func newEmptyDB(ctx context.Context) (*sql.DB, error) {
+	fmt.Println("new Db")
```

Comment:
Are these print calls just for debugging, and can therefore be removed?

Reply:
Sure